services/flow-repository/app/utils/publish.js

```diff
@@ -0,0 +1,23 @@
+const config = require('../config/index');
+const log = require('../config/logger');
+const { EventBus, RabbitMqTransport, Event } = require('../../../../lib/event-bus');
+
+async function publish(ev) {
```

Comment:
My vision was, that an application should connect to the Event Bus on start and keep this connection till the termination. Therefore I find this code weird.

Reply:
You're entirely correct. Adjusted to build a queue during startup and maintain it.